src/cli/keepassxc-cli.1

```diff
@@ -9,13 +9,19 @@ keepassxc-cli \- command line interface for the \fBKeePassXC\fP password manager
 .I command
 
 .SH DESCRIPTION
-\fBkeepassxc-cli\fP is the command line interface for the \fBKeePassXC\fP password manager. It provides the ability of listing the entries of a database, displaying the contents of an entry and many more, directly from the command line.
+\fBkeepassxc-cli\fP is the command line interface for the \fBKeePassXC\fP password manager. It provides the ability to query and modify the entries of a KeePassXC database, directly from the command line.
```

Comment:
`...entries of a KeePassXC database...` 

Personally I will change this to KeePass database since that's the file format